test/framework/src/main/java/org/elasticsearch/AnalysisFactoryTestCase.java

```diff
@@ -178,7 +175,7 @@ private static String toCamelCase(String s) {
         .put("apostrophe",                ApostropheFilterFactory.class)
         .put("arabicnormalization",       ArabicNormalizationFilterFactory.class)
         .put("arabicstem",                ArabicStemTokenFilterFactory.class)
-        .put("asciifolding",              ASCIIFoldingTokenFilterFactory.class)
+        .put("asciifolding",              Void.class)  // TODO remove this when core no longer depends on analysis-common
```

Comment:
The test is written in such a way that it'll fail if there are any unmapped analysis components. It is our reminder to expose whatever is in Lucene. So we can't remove it. Instead we have to pretend we're intentionally not exposing it which we do by marking it as exposed by `Void`. At least, that is the pattern the test uses. It is a little funny, but the TODO notes that it should go away when we finally drop analyzer's dependency from core.